code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -624,6 +619,53 @@
 	var/footstep = 1
 	var/datum/component/mobhook
 
+/obj/item/clothing/suit/space/hardsuit/ancient/mason
+	name = "M.A.S.O.N RIG"
+	desc = "The Multy Advanced Sencer Of Networks RIG type hardsuit. With shining plates to protect against not only rads storms, SM meltdowns, acid proof coating this hulking hardsuit has huds built into its visors.\
```

Comment:
Can we please not have a badly written novel for a description?

desc = The Multi-Augmented Severe Operations Networked Resource Integration Gear is an man-portable tank designed for extreme environmental situations. It is excessively bulky, but rated for all but the most atomic of hazards. The specialized armor is surprisingly weak to conventional weaponry.